support/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# try to fetch the latest version from github
+wget -O support.sh.new https://raw.githubusercontent.com/docker/libnetwork/master/support/support.sh
+
+if [ "$?" -eq "0" ]; then
+  mv support.sh.new support.sh
+  chmod +x support.sh
+else
+  echo "issue fetching the latest support.sh, will use the container version"
+fi
+
+echo "run the support script"
+./support.sh
```

Comment:
Minor nit:  assumes that the `support.sh` is in the same directory as this `run.sh` script is run from.  More robust to just assume that support.sh is in the runtime path?

Reply:
The Dockerfile uses as workdir `/bin`
from docs: 
```
The WORKDIR instruction sets the working directory for any RUN, CMD, ENTRYPOINT, COPY and ADD instructions that follow it in the Dockerfile
```
so the script is running with path /bin

Reply:
ah, that's true.  ok.